bitnami/kafka/templates/jmx-configmap.yaml

```diff
@@ -16,6 +16,9 @@ data:
   jmx-kafka-prometheus.yml: |-
     {{- include "common.tplvalues.render" ( dict "value" .Values.metrics.jmx.config "context" $ ) | nindent 4 }}
     rules:
+      {{- if .Values.metrics.jmx.extrarules }}
```

Comment:
semantically, I would put "extra" stuff at the end

Reply:
I agree 👍

Reply:
I moved the logic to the end. thanks